sbitx.c

```diff
@@ -94,6 +93,18 @@ static int bridge_compensation = 100;
 static double voice_clip_level = 0.04;
 static int in_calibration = 1; // this turns off alc, clipping et al
 static double ssb_val = 1.0;  // W9JES
+extern void check_r1_volume();//Volume control normalization W2JON
```

Comment:
Takes the Static int rx_vol and recalculates it based on the current setting of the r1:volume control